common/config/rush/experiments.json

```diff
@@ -55,6 +55,12 @@
    */
   "phasedCommands": true
 
+  /**
+   * If true, rush install or rush update implicitly specify --ignore-scripts during pnpm install,
+   * and run install lifecycle scripts by pnpm rebuild --pending after pnpm install successfully.
+   */
+  /* "deferredInstallationScripts": true,
```

Comment:
It works because it's commented out. I'm going to temporarily delete this, since our convention is to introduce these lines when upgrading Rush itself.  For this PR the appropriate place to add it is in the the `rush-init` template.

Reply:
Fixed